### PR TITLE
Fee rate median and max mining fee fixes

### PR DESCRIPTION
**Pull request by komret**
## Description

This PR fixes some bugs I made in #7782 and and #7743 😅 

fa5d5e82b4da6bef83ab0ca65e19befcc1c6eca0: Default max mining fee is derived from weekly fee rate median. I originally assumed the fee rate median will not be used otherwise, so I only saved the derived `maxMiningFee`. This assumption was wrong because the median should be used in `selectMinAllowedInputWithFee` and `MaxMiningFeeSetup` range gradient. This commit replaces `maxMiningFee` with `weeklyFeeRateMedian` in `CoinjoinStatusEvent`.

b509b1a51b3e7c04628bb02787b386652dd106af: This commit applies the changes made in `@trezor/coinjoin` to `@trezor/suite`, thus fixing the calculation in `selectMinAllowedInputWithFee`. It also makes sure to always round the `maxFeePerVbyte` value so that it cannot result in fractions of satoshis. Also some minor refactoring.

f3fd94f755ea65a1692949af077bad606acbabd6: Improved gradient percentage calculation in `MaxMiningFeeSetup`.

24272048ff32a0533fcd233985b542cad3abaa06: Minor layout change to align cards between tabs in coinjoin account.

**QA:** This is mostly code improvement subtly affecting minimum UTXO amount for coinjoin and range styles. Starting coinjoin is still disabled with `Amounts are too small for coinjoin` when each non-private UTXO is too small, i.e. below 5000 sat + fee. Previously, the default max mining fee was used as a fee estimate - this was now replaced with fee rate median, the difference is usually a few hundred sats.

### CODE REVIEW

**Comment by komret** [comment]
I'll wait for #7902 to be merged as it is a prerequisite.

File: packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx [hunk 1]
```diff
@@ -117,16 +118,17 @@ export const CoinjoinConfirmation = ({ account }: CoinjoinConfirmationProps) =>
...
 
     const dispatch = useDispatch();
 
     const { coinjoinSessionBlockedMessage, isCoinjoinSessionBlocked } = useCoinjoinSessionBlockers(
         account.key,
     );
 
-    const anonymitySet = account.addresses?.anonymitySet;
-
-    if (!coinjoinClient || !targetAnonymity || !anonymitySet || !coinjoinAccount) {
```
Comment (matejkriz) [inline packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx:129]: why this change?

Comment (komret) [reply packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx:129]: I noticed that `anonymitySet` was used in this component in the past, but not anymore.

Comment (matejkriz) [reply packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx:129]: @dahaca is it ok to remove this condition from here?

**Review by matejkriz — APPROVED** [review]
Looks good, works as expected 👌

**Comment by STew790** [comment]
**QA OK**
I tried to send 1 `sat` lower then the minimum on BTCtest and it wouldn't let me CJ. After sending UTXO with the exact minimum, I was able to start the CJ.

image

image

**Info:**
 - Suite version: desktop 23.4.0 (de59a07c5ee7fcb9947dd4373194548ba186a2a2)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.4.0 Chrome/110.0.5481.104 Electron/23.1.1 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model 1 1.12.1 Universal (revision 1eb0eb9d91b092e571aac63db4ebff2a07fd8a1f)
 - Transport: BridgeTransport 2.0.32
